module/observationSubmissions/helper.js

```diff
@@ -126,11 +127,24 @@ module.exports = class ObservationSubmissionsHelper {
                   ["name","description"],
               );
 
+              programDocument = programDocument.data.data;
```

Comment:
@borkarsaish65 suggesting to handle the errors
const programData = programResponse?.data?.data;
            if (programData?.length) {
                observationSubmissionsDocument.programInfo = programData[0];
            }